src/Components/Contributors/Contributor.tsx

```diff
@@ -18,6 +19,11 @@ const StyledVerifiedBadge = styled(VerifiedBadge)`
   }
 `;
 
+const StyledAlert = styled(Alert)`
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+`;
+
```

Comment:
Er ikke fan av alerts. Går det an å vise heller frem hvilke to som er dubletter i tabellen?

Reply:
testet først med en rutine som sjekket hele lista og lagde en visning på slutten, men syntes det passet med feilmelding pr post. Dessuten  tenkte jeg det holdt med en enkel warning - så får de søke selv.

Reply:
fikset peker til hvilke forfattere det er snakk om. men alerten vil jeg ha :)